Makefile

```diff
@@ -50,9 +45,7 @@ NUNIT_CONSOLE = packages/NUnit.ConsoleRunner.3.7.0/tools/nunit3-console.exe
 BUILD_PROPS = bin/Build$(CONFIGURATION)/JdkInfo.props bin/Build$(CONFIGURATION)/MonoInfo.props
 
 all: $(BUILD_PROPS)  src/Java.Runtime.Environment/Java.Runtime.Environment.dll.config \
-		$(PACKAGES) $(DEPENDENCIES) $(TESTS) $(XA_INTEGRATION_OUTPUTS)
-
-xa-all: $(PACKAGES) $(XA_INTEGRATION_OUTPUTS)
+		$(PACKAGES) $(DEPENDENCIES) $(TESTS) bin/$(CONFIGURATION)/Java.Interop.dll
```

Comment:
Why do we need to add `bin/$(CONFIGURATION)/Java.Interop.dll` to the dependencies list?

Additionally, there's no target to build `bin/$(CONFIGURATION)/Java.Interop.dll`, so how's this work?

Reply:
Whoops, yes, I messed this up.